cocos/2d/CCAutoPolygon.cpp

```diff
@@ -87,6 +87,17 @@ void PolygonInfo::setQuad(V3F_C4B_T2F_Quad *quad)
     triangles.verts = (V3F_C4B_T2F*)quad;
 }
 
+void PolygonInfo::setTriangles(TrianglesCommand::Triangles other)
+{
+    this->releaseVertsAndIndices();
+    isVertsOwner = false;
+    
+    this->triangles.vertCount = other.vertCount;
+    this->triangles.indexCount = other.indexCount;
+    this->triangles.verts = other.verts;
```

Comment:
`verts` is a pointer, It is not safe to do it like this if `other`'s `verts` is deleted.